src/components/con/editions/EditionCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useStaticQuery, graphql, Link } from 'gatsby';
+import { GatsbyImage, getImage } from 'gatsby-plugin-image';
+import { Edition } from '../types';
+
+interface EditionCardProps {
+  edition: Edition;
+  size: 'small' | 'big';
+}
+
+const EditionCard: React.ComponentType<EditionCardProps> = ({ edition, size = 'big' }) => {
+  const data = useStaticQuery(graphql`
+    query {
+      allFile(filter: { sourceInstanceName: { eq: "editionsImages" } }) {
+        nodes {
+          name
+          small: childImageSharp {
+            gatsbyImageData(width: 240, placeholder: DOMINANT_COLOR)
+          }
+          big: childImageSharp {
+            gatsbyImageData(width: 400, placeholder: DOMINANT_COLOR)
+          }
+        }
+      }
+    }
+  `);
+
+  const images = data.allFile.nodes.filter((imageData) => imageData.name === edition.image)?.[0];
```

Comment:
find method instead of filter if you need the first element found? (if i'm understanding the logic here)